src/test/java/hudson/plugins/git/GitSCMTest.java

```diff
@@ -1655,7 +1655,6 @@ public void testCheckoutToDefaultLocalBranch_NULL() throws Exception {
        assertEquals("GIT_BRANCH", "origin/master", getEnvVars(project).get(GitSCM.GIT_BRANCH));
        assertEquals("GIT_LOCAL_BRANCH", "master", getEnvVars(project).get(GitSCM.GIT_LOCAL_BRANCH));
     }
-
```

Comment:
Please put back this blank line.  Whitespace changes in the diffs are a distraction and can make other pull requests fails with an unnecessary conflict.